frontend-mobile_pt.md

```diff
@@ -0,0 +1,78 @@
+# Desafio de Frontend/Mobile
+
+## Sobre o desafio 
+
+Nós usamos GraphQL aqui(dê uma olhada em https://www.graphql.com/guides/), então essa será a forma
+que você receberá os dados (soa bem, certo?).
+
+### O que você deve desenvolver?
+É simples. O usuário precisa escrever o seu endereço dentro de um campo e ser redirecionado
+para a página de listagem de produtos. O seu trabalho como frontend é desenvolver 2 telas simples consumindo e enviando dados para nossa API.
+
+Você pode checar um exemplo desse fluxo aqui: https://ze.delivery/ (nosso ambiente de produção)
+
+### Visual
+Nós queremos que você se sinta livre para desenvolver o modelo do jeito que você quiser, nós não temos 'wireframes' ou algo do tipo para essa tarefa. Você irá criar duas telas:
+  - Página Home:
+    - Precisa ter um cabeçalho, um rodapé e um input para o usuário preencher seu endereço
+  - Página Products:
+    - Nós queremos ver os produtos em 'cards', com o título do produto, preço e botões para adicionar e remover o produto.
+
+### Sobre a API
+O link da API que você irá utilizar: https://api.code-challenge.ze.delivery/public/graphql 
+(Se você tentar abrir isso em um navegador irá aparecer "Missing Authentication Token", você precisa utilizar no código)
+  - Página Home:
+    - Para encontrar a latitude e longitude do endereço, você pode utilizar a API do Google Maps ou utilizar qualquer outra API de Geocoder que você quiser.
+    - Utilize o seguinte endereço para encontrar o distribuidor (PoC): Rua Américo Brasiliense, São Paulo.
+    - A Query necessária para obter o distribuidor correto (obtém o primeiro distribuidor):
+        - Query: https://bit.ly/32Gb2Qi
+        - Argumentos:
+          - Algorithm:   "NEAREST".
+          - Lat: Envie como String.
+          - Long: Envie como String.
+          - Now: A Data atual formatada.
+  - Página Products:
+      - A Query necessária para obter os produtos
+        - Query: https://bit.ly/2N3JQ7k
+        - Argumentos:
+          - Id: id do distribuidor recebido na query anterior (PocSearch)
+          - Search: se "" (vazio), não filtra por 'search'.
+          - CategoryId: se null, não filtra por 'category'.
+      - A Query necessária para obter a lista de categorias
+        - Query: https://bit.ly/2pMG8Xy
+
+
+### Requerimentos técnicos (para as duas posições):
+- Você precisa criar um repositório no GitHub para hospedar seu código
+  - Usuários do Github:
+      - @ze-engineering-code-challenge
+- Evite usar 'scaffolds' (Create React App etc.), nós queremos ver como você cria a estrutura do projeto
+- Mantenha simples, não há necessidade de coisas extravagantes :)
+- TESTES 💛 Nós realmente amamos testar nosso código e suas característica aqui!
+
+#### Se você estiver fazendo para a posição de frontend:
+- Você não deve usar Framework CSS (Bootstrap, Tailwind, Bulma, PureCSS etc.)
+- React é um a mais, mas você pode usar o que quiser =)
```

Comment:
```suggestion
- React é um plus, mas você pode usar o que quiser =)
```

Reply:
Se quiser deixar como estrangeirismo, é só por o itálico que fica legal